pkg/transport/generic_transport_controller.go

```diff
@@ -140,18 +147,44 @@ type genericTransportController struct {
 	wrappedObjectGVR     schema.GroupVersionResource
 }
 
-// enqueueEdgePlacementDecision takes an EdgePlacementDecision resource and converts it into a namespace/name
-// string which is then put onto the work queue. This method should *not* be passed resources of any type other than EdgePlacementDecision.
+// enqueueEdgePlacementDecision takes an EdgePlacementDecision resource and
+// converts it into a namespace/name string which is put onto the workqueue.
+// This func *shouldn't* handle any resource other than EdgePlacementDecision.
 func (c *genericTransportController) enqueueEdgePlacementDecision(obj interface{}) {
 	var key string
 	var err error
 	if key, err = cache.MetaNamespaceKeyFunc(obj); err != nil {
-		c.logger.Error(err, "failed to enqueue EdgePlacementDecision object")
+		c.logger.Error(err, "failed to extract key from EdgePlacementDecision object")
```

Comment:
I think this change does make the error more accurate, but for debugging clarity later (or log clarity overall) - it should probably also still include that the enqueuing has failed, since no other error level around calling this function reports it.

Reply:
reverted to previous log message.
merging the PR